src/Report.php

```diff
@@ -8,6 +8,7 @@
 
 abstract class Report
 {
+    public $filters;
```

Comment:
i esu?

Reply:
Els he tret del Report.php normal i els he ficat aquí dins. No estaven definits

Reply:
pero quina diferencia hi ha? perque ho necessita el report? a retail no hi es pas

Reply:
O sigui, utilitzem $this->filters a dins de la classe, però no la teníem declarada!